lib/queue.js

```diff
@@ -41,21 +30,79 @@ function compare (a, b) {
     return matches;
 }
 
+/**
+ * Creates a new queue
+ * @class
+ */
 var FetchQueue = function() {
-    this.oldestUnfetchedIndex = 0;
```

Comment:
I made `referrerQueueItem`, `completeCache` and `scanIndex` non-enumerable properties and renamed them to start with an underscore. We don't use them when defrosting a queue, so it makes sense to not include them in the frozen JSON file either (and this is a good way of avoiding that)